test/parallel/parallel.status

```diff
@@ -0,0 +1,27 @@
+prefix parallel
+
+# To mark a test as flaky, list the test name in the appropriate section
+# below, without ".js", followed by ": PASS,FLAKY". Example:
+# sample-test                       : PASS,FLAKY
+
+[true] # This section applies to all platforms
```

Comment:
I'd add https://github.com/nodejs/node/issues/1100 since it can fail on virtually any platform at random.